src/ci/docker/x86_64-gnu-tools/checktools.sh

```diff
@@ -57,17 +57,35 @@ verify_status() {
     fi
 }
 
+check_dispatch() {
+    if [ "$1" = submodule_changed ]; then
+        # ignore $2 (branch id)
+        verify_status $3 $4
+    else
+        if [ "$2" = beta ]; then
```

Comment:
This could be `elif`

Reply:
I think "beta" also here may want to be "beta_required"?

Reply:
so close, I tried `elsif`

Reply:
the "beta" here is the second arg to all function calls in status_check. It denotes the branch, not the kind of check. I should probably add more comments